src/ert/gui/ertwidgets/analysismodulevariablespanel.py

```diff
@@ -90,7 +101,8 @@ def __init__(self, analysis_module: AnalysisModule, ensemble_size: int):
 
             localization_frame = QFrame()
             localization_frame.setLayout(QHBoxLayout())
-            localization_frame.layout().setContentsMargins(0, 0, 0, 0)
+            lf_layout = cast(QLayout, localization_frame.layout())
```

Comment:
(nitpick) Why cast instead of assert in this case?

Reply:
because cast is free and we know that it is not none. plus we can in-line the cast :)